lightapi/jwt_custom.py

```
@@ -0,0 +1,61 @@
+    else:
+        raise ValueError("Unsupported algorithm")
+        
+    encoded_signature = b64_encode(signature)
+    
+    return f"{encoded_header}.{encoded_payload}.{encoded_signature}"
+
+def jwt_decode(token: str, secret: str, algorithms: list[str] = ["HS256"]) -> dict:
+    try:
+        encoded_header, encoded_payload, encoded_signature = token.split(".")
+    except ValueError:
+        raise ValueError("Invalid token")
```

Ruff (0.14.1)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)